submarine-workbench/workbench-server/src/main/java/org/apache/submarine/server/SubmarineConfiguration.java

```diff
@@ -374,7 +374,7 @@ public boolean getBoolean(String envName, String propertyName, boolean defaultVa
         "failOverReadOnly=false&amp;zeroDateTimeBehavior=convertToNull&amp;useSSL=false"),
     JDBC_USERNAME("jdbc.username", "submarine"),
     JDBC_PASSWORD("jdbc.password", "password"),
-    WORKBENCH_WEB_WAR("workbench.web.war", "submarine-workbench/workbench-web/dist");
+    WORKBENCH_WEB_WAR("workbench.web.war", "../workbench/workbench-web.war");
```

Comment:
Don't Modify `WORKBENCH_WEB_WAR` default value.
If change `WORKBENCH_WEB_WAR` default value. then cannot run/debug WorkbenchServer directly in IDEA directly, because WorkbenchServer module not contain `workbench-web.war`.

Reply:
Run the workbench Server in the formal environment, which can be run by configuring the war file path in `submairne-site.xml`.

Reply:
Got you